HerPublicWebsite.BusinessLogic/ExternalServices/EpbEpc/EpbEpcApi.cs

```diff
@@ -1,11 +1,80 @@
-﻿namespace HerPublicWebsite.BusinessLogic.ExternalServices.EpbEpc
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+using HerPublicWebsite.BusinessLogic.ExternalServices.Common;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System.Net.Http.Headers;
+
+namespace HerPublicWebsite.BusinessLogic.ExternalServices.EpbEpc
 {
+
+    public record struct EpcAddress
```

Comment:
Is there a particular reason these are `struct`s?

Reply:
Nope! Should definitely be classes